src/main/kotlin/org/jitsi/jicofo/codec/CodecUtil.kt

```diff
@@ -145,7 +147,8 @@ class CodecUtil {
                 }
                 if (config.opus.useInbandFec()) {
                     // fmtp:111 useinbandfec=1
-                    opus.addParameterExtension("useinbandfec", "1")
+                    val fecval = addedRed.compareTo(true).toString()
```

Comment:
If we've added RED then we add `useinbandfec=false`, but if `opus.useInbandFec` is false we don't add an extensin at all. Is this intentional? If not, you could simplify to just `if (config.opus.useInbandFec() && !config.opus.red.enabled())`

Reply:
> If we've added RED then we add `useinbandfec=false`, but if `opus.useInbandFec` is false we don't add an extensin at all. Is this intentional? If not, you could simplify to just `if (config.opus.useInbandFec() && !config.opus.red.enabled())`

Yeah it is kind of intentional. This results in `useinbandfec=0`. I wanted to be explicit about it, rather than rely on absence of the `useinbandfec` parameter to do the right thing.
And I decided to not rely on the config option in case someone later changes the if condition for the RED block.
But if you think it is important to change to your version I'll update my commit.